docs/guides/publisher-client-side.md

```diff
@@ -55,14 +60,16 @@ Only root-level domains are required for account setup. For example, if you're g
 
 ## Add UID2 SDK For JavaScript to Your Site
 
+{**GWH_SW_02: The title indicates that this is a step, so I'd expect some instruction here. Did we miss anything? Like, add this code to your site? (where). Or any other sort of guidance, like if this add this bit of code, if some other scenario add a different bit?**}
```

Comment:
_{**GWH_SW_02: The title indicates that this is a step, so I'd expect some instruction here. Did we miss anything? Like, add this code to your site? (where). Or any other sort of guidance, like if this add this bit of code, if some other scenario add a different bit?**}_

rephrased to say the code below is something pubs need to copy but a more thorough example will be provided further below.

Reply:
@sunnywu perfect, thx!

Reply:
@genwhittTTD i have reworded the sentences that the code snippet here is just for overview understand and explicitly tell the reader to copy the Example codes in **_Example Integration Code and When to Pass DII to the UID2 SDK section_**. Think this is good enough for now but feel free to edit my sentences further.